docs/guide/installation.md

```diff
@@ -9,7 +9,7 @@ If you have the previous `vue-cli` (1.x or 2.x) package installed globally, you
 Vue CLI requires [Node.js](https://nodejs.org/) version 8.9 or above (8.11.0+ recommended). You can manage multiple versions of Node on the same machine with [nvm](https://github.com/creationix/nvm) or [nvm-windows](https://github.com/coreybutler/nvm-windows).
 :::
 
-To install the new package, use one of those commands:
+To install the new package, use one of those commands (you need administrator privileges to install globally):
```

Comment:
May need.
If your npm is installed by a node version manager (e.g. n or nvm), then root privilege is no required.